suites/pacific/rgw/tier_1_rgw_cephadm.yaml

```diff
@@ -0,0 +1,125 @@
+# RGW daemons configured with multiple realm, along with zones and zonegroups. Here,
+# the primary interface used would be CLI of CephADM.
+tests:
+
+  # Cluster deployment stage
+
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
+      abort-on-fail: true
+      config:
+        verify_cluster_health: true
+        steps:
+          - config:
+              command: bootstrap
+              service: cephadm
+              args:
+                registry-url: registry.redhat.io
+                mon-ip: node1
+                orphan-initial-daemons: true
+                skip-dashboard: true
+          - config:
+              command: add_hosts
+              service: host
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              command: apply
+              service: mgr
+              args:
+                placement:
+                  label: mgr
+          - config:
+              command: apply
+              service: mon
+              args:
+                placement:
+                  label: mon
+          - config:
+              command: apply
+              service: osd
+              args:
+                all-available-devices: true
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.1
+              args:
+                placement:
+                  nodes:
+                    - node5
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.2
+              args:
+                port: 81
+                placement:
+                  nodes:
+                    - node5
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.3
+              args:
+                placement:
+                  nodes:
+                    - node6
+          - config:
+              command: apply
+              service: rgw
+              pos_args:
+                - rgw.4
+              args:
+                port: 81
+                placement:
+                  nodes:
+                    - node6
+      desc: RHCS cluster deployment using cephadm.
+      destroy-cluster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      config:
+        command: add
+        id: client.1
+        node: node7
+        install_packages:
+          - ceph-common
+        copy_admin_keyring: true
+      desc: Configure the RGW client system
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      abort-on-fail: true
+      config:
+        cephadm: true
+        commands:
```

Comment:
Do you think we need separate module for radosgw-admin command operation?

Where we can take RGW command execution individually as test or calling all this commands from test_module(However we required).